config.py

```diff
@@ -43,7 +43,9 @@ class Config:
     DATA_VERSION = os.getenv('OPENTARGETS_DATA_VERSION', '17.04')
 
     # easier to use with less parameters
-    ES_PREFIX = partial(prefix=DATA_VERSION, ini=ES_CUSTOM_IDXS_INI)
+    ES_PREFIX = partial(prefix_or_custom_idx,
+                        prefix=DATA_VERSION,
+                        ini=ES_CUSTOM_IDXS_INI)
```

Comment:
for it to work with fern.opentargets.io, we need to be able to pass these as multiple ENV vars. we can keep the ini file but one must be able to override it with ENVs so that we can declare them in `cd.yaml`, similarly to how it's spec'd in #69